docker/build/edxapp/Dockerfile

```
@@ -20,15 +20,19 @@ COPY docker/build/edxapp/ansible_overrides.yml /
 COPY docker/build/edxapp/devstack.yml /
 COPY docker/build/devstack/ansible_overrides.yml /devstack/ansible_overrides.yml
 
-ARG OPENEDX_RELEASE=master
+ARG OPENEDX_RELEASE=appsembler/tahoe/master
+ARG APPSEMBLER_PLATFORM_REPO=https://github.com/appsembler/edx-platform.git
```

I think docker now supports "checkout out git code as the host" as opposed to "as the container".

If that's possible, it would be awesome!

```suggestion
ARG APPSEMBLER_PLATFORM_REPO=git@github.com:appsembler/edx-platform.git
```

---

I'll remove this one. nvm.